RELEASE.md

```diff
@@ -0,0 +1,113 @@
+
+* Models using custom backends which implement auto-complete configuration, can 
+  be loaded without explicit config.pbtxt file if they are named in form 
+  `<model_name>.<backend_name>`.
+
+* Users can specify a maximum memory limit when loading models onto the GPU  
+  with the new 
+  [--model-load-gpu-limit](https://github.com/triton-inference-server/server/blob/b3d7a3375e7adb1341724c0ac34661b4cde23cd2/src/main.cc#L629-L635)
+  tritonserver option and the 
+  [TRITONSERVER_ServerOptionsSetModelLoadDeviceLimit](https://github.com/triton-inference-server/core/blob/c9cd6630ecb04bb26e2110cd65a37f23aec8153b/include/triton/core/tritonserver.h#L1861-L1872) C API function
+
+* Added new documentation, 
+  [Performance Tuning](https://github.com/triton-inference-server/server/blob/main/docs/performance_tuning.md), with a step by step guide to optimize models for 
+  production
+
+* From this release onwards Triton will default to 
+  [TensorFlow version 2.X.](https://github.com/triton-inference-server/tensorflow_backend/tree/main#--backend-configtensorflowversionint) 
+  TensorFlow version 1.X can still be manually specified via backend config.
+
+* PyTorch backend has improved performance by using a separate CUDA Stream for 
+  each model instance when the instance kind is GPU.
+
+* Refer to the 22.08 column of the 
+  [Frameworks Support Matrix](https://docs.nvidia.com/deeplearning/frameworks/support-matrix/index.html) 
+  for container image versions on which the 22.08 inference server container is 
+  based.
+
+* Model Analyzer's profile subcommand now analyzes the results after Profile is 
+  completed. Usage of the Analyze subcommand is deprecated. See 
+  [MA documentation](https://github.com/triton-inference-server/model_analyzer/blob/main/docs/cli.md#subcommand-profile) 
+  for further details.
+
+## Known Issues
+
+* There is no Jetpack release for 22.08, the latest release is 22.07.
+
+* Auto-complete may cause an increase in server start time. To avoid a start 
+  time increase, users can provide the full model configuration and launch the 
+  server with `--disable-auto-complete-config`.
+
+* When auto-completing some model configs, backends may generate a model config 
+  even though there is not enough metadata (ex. Graphdef models for TensorFlow 
+  Backend). The user will see the model successfully load but fail to inference. 
+  In this case the user should provide the full model configuration for these 
+  models or use the `--disable-auto-complete-config` CLI option to show which 
+  models fail to load.
+
+* Auto-complete does not support PyTorch models due to lack of metadata in the 
+  model. It can only verify that the number of inputs and the input names 
+  matches what is specified in the model configuration. There is no model 
+  metadata about the number of outputs and datatypes. Related PyTorch bug: 
+  https://github.com/pytorch/pytorch/issues/38273
+
```

Comment:
Add new bullet point as documentation updated:

>Auto-complete is not supported in the OpenVINO backend

Reply:
Should be resolved - see below